sympy/core/tests/test_evaluate.py

```diff
@@ -6,7 +6,13 @@
 def test_add():
     with evaluate(False):
         assert pretty(oo-oo) is not nan
```

Comment:
Better be like this
```py
p = oo - oo
assert isinstance(p, And) and p.args == (oo, oo)
```
It is not about `pretty` anymore.

Reply:
Sure, I'll update. But why did that linter fail for the first commit?

Reply:
Sometimes code editors add extra whitespaces and sympy's code quality doesn't allow that. But it is easy to handle by testing it before each push(its mostly the whitespaces).

Reply:
> Sometimes code editors add extra whitespaces and sympy's code quality doesn't allow that.

Yeah, I got that last time.. but I'm talking about the first commit I added today. View config shows this: 

This job was cancelled because the "Auto Cancellation" feature is currently enabled, and a more recent build (#37724) for pull request #18109 came in while this job was waiting to be processed.

Reply:
The tests are more than a linter. And since you pushed more commits, the previous ones become outdated. Hence no point in testing them. A new build is made and testing is done.

Reply:
Okay! Just one more doubt before I update the test file.  For the rest of the cases, I just need to update `p.args == (oo, oo)` to let's say  `p.args == (oo, -5)` or `p.args == (5, -oo)  ` 

> It is not about pretty anymore.

Can you explain how?  because I didn't quite get the new way of adding the test

Reply:
For the rest of the test cases, execute the code in `with evaluate(False)` and find `p.args` for each expression.

> Can you explain how?

Here printing is not the problem, the additions were getting auto-evaluated. We are currently checking that result is still not evaluated (is an `And` object) and has same `args`

Reply:
p = oo - oo
But isinstance(p, And) always returns False.

Reply:
I think that should be `assert (isinstance(p, And) == False) and p.args == (oo,  -oo)` for p = oo - oo
Please correct me if I'm wrong :)

Reply:
My bad, it's `Add` (instead of `And`). Too much working with booleans :)

Reply:
Aha, Okay. If any changes are still required, please suggest, otherwise, I think the request is ready to be merged after the green tick.